dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -459,7 +465,7 @@ CDefaultContextMenu::EnumerateDynamicContextHandlerForKey(HKEY hRootKey)
             }
         }
 
-        hr = LoadDynamicContextMenuHandler(hKey, clsid);
+        hr = LoadDynamicContextMenuHandler(hRootKey, clsid);
```

Comment:
Is this a real bug fix, irrespective of the 7zip extension discussion we had in the main PR comments?

Reply:
It has nothing to do with 7-zip, Windows passes the progid registry key, not `..\progid\shellex\{guid}`.